Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Hendrik Ranocha <mail@ranocha.de> and contributors"]
 version = "2.19.2"
 
 [deps]
+LaTeXStrings = "b964fa9f-0449-5b57-a5c2-d3ea65f4040f"
```

Comment:
Could you please add LaTeXStrings to the `[compat]` section below, too? Just insert  the stable version number you're using locally as done for the other packages.

This is required to register a new version of RootedTrees.jl.